.github/ISSUE_TEMPLATE/doc-issue.md

```diff
@@ -4,15 +4,47 @@ about: Create an issue to help us improve
 
 ---
 
-**Before you enter an issue**
+## Before you enter an issue
 
-- If the issue is a simple typo or similar correction, you can just submit a PR. See [the contributor guide](https://docs.microsoft.com/contribute/#quick-edits-to-existing-documents) for instructions, if needed.
-- If you are are struggling with a tutorial, check your code against the completed sample for differences.
-- If the issue is a general support question rather than a docs issue, consider a forum site instead of a docs issue.
-- If the issue is relating to site design, create an issue at [MicrosoftDocs/Feedback](https://github.com/MicrosoftDocs/Feedback/Issues/new/choose)
+If the issue is:
 
-**Include version and OS information**
-1. Include the OS you're using.
-1. Include the version of .NET you're using. Include "Framework" or "Core" as well.
+- A simple typo or similar correction, you can submit a PR. See [the contributor guide](https://docs.microsoft.com/contribute/#quick-edits-to-existing-documents) for instructions, if needed.
+- A general support question rather than a docs issue, consider a forum site instead of a docs issue.
```

Comment:
I'd try to simplify this and be more direct.

Support questions (not doc problems) should be posted at a support forum.

Or are better directed - or get quicker response.